Content.Client/Lobby/UI/LobbyGui.xaml

```diff
@@ -13,7 +13,7 @@
     <Control>
         <!-- Parallax background -->
         <parallax:ParallaxControl />
-        <TextureRect Access="Public" Name = "Background" Stretch="Scale"/>
+        <TextureRect Access="Public" Name = "Background" Stretch="KeepAspectCentered"/>
```

Comment:
Maybe use `KeepAspectCovered` instead.  Then you won't ever see the space background which probably looks nicer.

Also if you do that then we can probably remove or hide the parallax control, since right now it's just wasting rendering time presumably.

Reply:
Sounds good.